Xamarin.Forms.Platform.Android/Renderers/SearchHandlerAppearanceTracker.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Runtime;
+using Android.Text;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public class SearchHandlerAppearanceTracker : IDisposable
+	{
+		SearchHandler _searchHandler;
+		bool _disposed;
+		AView _control;
+		EditText _editText;
+		InputTypes _inputType;
+		TextColorSwitcher _textColorSwitcher;
+		TextColorSwitcher _hintColorSwitcher;
+
+		public SearchHandlerAppearanceTracker(IShellSearchView searchView)
+		{
+			_searchHandler = searchView.SearchHandler;
+			_control = searchView.View;
+			_searchHandler.PropertyChanged += SearchHandlerPropertyChanged;
+			_editText = (_control as ViewGroup).GetChildrenOfType<EditText>().FirstOrDefault();
```

Comment:
> The alignment of the placeholder to center does not work in iOS
> ![image](https://user-images.githubusercontent.com/27482193/55626583-92322380-57b4-11e9-8026-7dedc50cfc42.png)

also the orange is not the same.

Reply:
Color shades can be different because emulators work on different computers. I think that on the Mac always had more accurate color.
Here green is also different.